src/simpla-admin.html

```diff
@@ -89,11 +112,11 @@
 
           /**
            * Store of Simpla observers
-           * @type {Array}
+           * @type {Object}
            */
           _simplaObservers: {
-            type: Array,
-            value: []
+            type: Object,
+            value: {}
```

Comment:
Not a big deal given this is a singleton, but best practice, this should be a function, so `() => ({})`